searchtool/views.py

```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
```

Comment:
Is this file needed?

Reply:
This file will be needed in the future when defining how users will navigate through the search tool